WMass/python/plotter/w-helicity-13TeV/mergeCardComponentsAbsY.py

```diff
@@ -262,7 +262,7 @@ def putEffStatHistos(infile,regexp,charge):
     
         ## prepare the relevant files. only the datacards and the correct charge
         allfiles = [os.path.join(dp, f) for dp, dn, fn in os.walk(options.inputdir) for f in fn if (f.endswith('.card.txt') or f.endswith('.input.root'))]
-        files = [f for f in allfiles if charge in f and not re.match('.*_pdf.*|.*_muR.*|.*_muF.*|.*alphaS.*|.*wptSlope.*|.*mW.*',f) and f.endswith('.card.txt')]
+        files = [f for f in allfiles if charge in f and not re.match('.*_pdf.*|.*_muR.*|.*_muF.*|.*alphaS.*|.*wptSlope.*|.*mW.*|.*ErfPar\dEffStat.*',f) and f.endswith('.card.txt')]
```

Comment:
@emanueledimarco added back otherwise it fails if there are still the txt / root files from the ErfPar in the directories. Doesn't hurt if they are not there, like when we reweight post-fact with @mdunser code